corso_website_fe/src/Pages/CompleteOrdersPage.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import OrderTile from "../Components/OrderTile";
+import CustomPagination from "../Components/Pagination";
+import { Order } from "../Components/OrderTile";
+import NavigationBar from "../Components/NavigationBar";
+
+function CompletedOrdersPage() {
+    const [orders, setOrders] = useState<Order[]>([]);
+    const [page, setPage] = useState(1);
+    const [pageSize, ] = useState(1);
+    const [totalPages, setTotalPages] = useState(1);
+
+    const fetchOrders = async (pageNumber: number, size: number) => {
+        try {
+            const responseTotalOrders = await axios.get(`http://localhost:8080/api/v1/corso/orders?status=completed&pageSize=0&offset=0`);
+            const nbOrdersTotal = responseTotalOrders.data.length;
+
+            const newTotalPages = Math.ceil(nbOrdersTotal / size);
+
+            const response = await axios.get(`http://localhost:8080/api/v1/corso/orders?status=completed&pageSize=${size}&offset=${(pageNumber - 1) * size}`);
+            console.log(response.data)
+            const responseData = response.data;
+
+            const newOrders = await Promise.all(responseData.map(async (order: any) => {
+                const serviceResponse = await axios.get("http://localhost:8080/api/v1/corso/services/" + order.serviceId);
```

Comment:
Just to make sure this is a different page from your previous PR ? Meaning that the get all order page is not yet created ?